template/package.json

```diff
@@ -38,8 +38,7 @@
     "webpack-dev-server": "^3.1.14"
   },
   "dependencies": {
-    "@leanix/reporting": "^0.3.0",
-    "jquery": "^3.3.1",
+    "@leanix/reporting": "^0.3.3",
     "lodash": "^4.17.11"
```

Comment:
Lodash is optional, but for this report we are using it anyways, right? :-)

Reply:
Right :)